Template and VF Components/QueryComponent.component

```diff
@@ -0,0 +1,28 @@
+<apex:component controller="QueryComponentController">
+    <apex:attribute name="inputValue" description="Attribute on the component." type="String" required="required" assignTo="{!inputQuery}"/>
+    <apex:pageBlock title="Query Results" id="table">
+        <apex:pageBlockSection >
```

Comment:
`<apex:pageMessages />` is still missing from this component

Reply:
I add this tage in QueryComponent file.